distributions/test/src/test/java/com/connexta/ion/replication/distributions/test/ReplicationITest.java

```diff
@@ -0,0 +1,144 @@
+package com.connexta.ion.replication.distributions.test;
+
+import static com.jayway.restassured.RestAssured.given;
+import static org.awaitility.Awaitility.await;
+import static org.junit.Assert.assertTrue;
+
+import com.google.common.collect.ImmutableMap;
+import com.jayway.restassured.path.xml.XmlPath;
+import java.util.HashSet;
+import java.util.Set;
+import java.util.UUID;
+import java.util.concurrent.Callable;
+import java.util.concurrent.TimeUnit;
+import org.junit.After;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.testcontainers.containers.output.Slf4jLogConsumer;
+
+// TODO: why does this need to end in "test" for the tests to run?
+public class ReplicationITest extends IntegrationTest {
+
+  private static final Logger LOGGER = LoggerFactory.getLogger(ReplicationITest.class);
+
+  private static final Slf4jLogConsumer logConsumer = new Slf4jLogConsumer(LOGGER);
+
+  private Set<String> idsCreated = new HashSet<>();
+
+  @BeforeClass
+  public static void beforeClass() {
+    replication.followOutput(logConsumer);
+    solrUrl = "http://" + solr.getContainerIpAddress() + ":" + solr.getFirstMappedPort();
+    sourceUrl = System.getenv("source.node");
+    destinationUrl = System.getenv("destination.node");
+    if (sourceUrl == null) {
+      sourceUrl = "https://ditto-1.phx.connexta.com:8993/services";
+    }
+    if (destinationUrl == null) {
+      destinationUrl = "https://ditto-2.phx.connexta.com:8993/services";
+    }
+  }
+
+  @After
+  public void cleanUp() {
+    idsCreated.forEach(this::deleteFromNodes);
+    clearAllSolrCores();
+  }
+
+  @Test
+  public void createXmlFile() throws Exception {
```

Comment:
❓ Should the name be more inline with the intent of the test case? `createXmlFile` seems very short unclear to me